src/HelloWorld.java

```diff
@@ -4,6 +4,7 @@ public static void main(String[] args) {
         System.out.println("Hello World !");
 		if(1) {
 		System.out.println("Woo");
+		System.out.println("Hello");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS106)